plugins/periskop-backend/api-report.md

```diff
@@ -0,0 +1,22 @@
+## API Report File for "@backstage/plugin-periskop-backend"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { Config } from '@backstage/config';
+import express from 'express';
+import { Logger as Logger_2 } from 'winston';
+
+// @public (undocumented)
```

Comment:
Hey good job in keeping this clean! A bit of a hidden thing, but if you want to ensure that no new errors are introduced in your api in the future, you can add yourselves [to this blocklist](https://github.com/backstage/backstage/blob/master/scripts/api-extractor.ts#L201).